src/Apps/Collections/Components/CollectionsGrid.tsx

```diff
@@ -41,13 +50,19 @@ export class CollectionsGrid extends Component<CollectionsGridProps> {
                 <Media at="xs">{index === 0 && <Separator />}</Media>
                 <Media greaterThan="xs">{index < 3 && <Separator />}</Media>
 
-                <Box py={2}>
-                  <EntityHeader
-                    href={`/collection/${collection.slug}`}
-                    imageUrl={imageUrl}
-                    name={collection.title}
-                  />
-                </Box>
+                <LinkWithoutBorder
+                  noUnderline
+                  color="black100"
+                  href={`/collection/${collection.slug}`}
+                >
+                  <Flex py={2} alignItems="center">
+                    <Avatar size="xs" src={imageUrl} />
+
+                    <Serif ml={1} size="4t">
+                      {collection.title}
+                    </Serif>
+                  </Flex>
+                </LinkWithoutBorder>
```

Comment:
I ended up replacing the entire `<EntityHeader>` with palette components. The `<Flex>` component makes things simpler here so it doesn't seem like re-using the `<EntityHeader>` component is necessary.

Also, it would be interesting to make a `<FlexLink>` component if it's a common pattern to wrap elements with an `a` tag:

```jsx
<FlexLink py={2} alignItems="center" href={`/collection/${collection.slug}`}>
  <Avatar size="xs" src={imageUrl} />
  <Serif ml={1} size="4t">
    {collection.title}
  </Serif>
</FlexLink>
```

Reply:
@yuki24 - We're already using Entity header in a number of places; by replacing it outright we're introducing drift. If anything, this should be moved into palette and the previous EntityHeader callsites should be updated.

Reply:
List of locations: 

<img width="404" alt="screen shot 2018-12-10 at 1 39 42 pm" src="https://user-images.githubusercontent.com/236943/49753343-151cf480-fc81-11e8-8c31-f98ebe09275e.png">